src/features/product/components/product-specifications-section/index.tsx

```
@@ -0,0 +1,137 @@
+  Sprout,
+  Droplet,
+  Shirt,
+  HandHeart,
+  Wind,
+  Component,
+  Layers,
+  ClipboardCheck,
+  ShieldPlus,
+  Tag,
+  Rainbow,
+  Infinity,
```

Biome (1.9.4)

[error] Do not shadow the global "Infinity" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)